api/controllers/method.js

```diff
@@ -31,16 +34,6 @@ const articleText = require("../../static-text/article-text.js");
 const methodText = require("../../static-text/method-text.js");
 const sharedFieldOptions = require("../helpers/shared-field-options.js");
 
-async function getEditStaticText(params) {
```

Comment:
this is function is still needed to gather the static field options to pass to the edit views. once the localization branch is merged in it will edited, but still needed.

Reply:
OK, there is no METHOD_EDIT_STATIC query for the database, so I will put this function back, just returning an empty object for now, and populate it when we merge localization branch later today.